specs/registry-auth.md

```diff
@@ -0,0 +1,143 @@
+# Registry Authentication
+
+Registry access is required for pulling the manifests of the images to be verified along with their signatures as well as other advanced operations. This documentation specifies how authentication to the remote registry works and how registry credentials are stored.
+
+
+## Communication Channel
+
+Although it is secure to transmit artifacts with their signatures via HTTP connections, it is RECOMMENDED to transmit via HTTPS connections for confidentiality and the authenticity of the remote server.

```

Comment:
Not sure why we say this, at least on public networks, HTTP is vulnerable to MITM attack where attacker can spoof and respond with valid looking descriptors and associated content.

Reply:
> Not sure why we say this, at least on public networks, HTTP is vulnerable to MITM attack where attacker can spoof and respond with valid looking descriptors and associated content.

This is exactly what **authenticity** means.

Reply:
```suggestion
Although it is secure to transmit artifacts with their signatures via HTTP connections (as a tampered artifact can be detected through signature verification), it is RECOMMENDED to transmit via HTTPS connections for confidentiality and the authenticity of the remote server.

```

Reply:
Updated.